### DISCUSSION TITLE
Export my-graphs list

### DISCUSSION THREAD

**Discussion by jared-hughes** [DesModder/DesModder#837] [Ideas]
Probably after "customize my-graphs view." 

What format? 
 - JSON
 - Markdown
 - CSV
 - ??

Should thumbnails be included, and if so, should they be inline data:// urls or included as png files in an images ZIP file?

**Comment by FabriceNeyret** [comment]
For backup or export to other tools, the best is JSON: standard, structured, and complete.
For printing or showing banks of equations, some Markdown or even html is better.
→ better to do both, either by setup choice, or as 2 different features.

Thumbnails: costly, and regeneretable, plus URL to desmos thumbnail can often be sufficient.
But  might make sense for standalone copy, + long term backup ( including events like Desmos end, or no longer backward compatible).
→ again, best is to let the choice, I would say.
Option: main file using the url thumbnail, and separate image file allowing to substitute
https://saved-work.desmos.com/calc_thumbs/production/  to some local directory.
